src/components/DocsCollection/SpokeDocsList.astro

```diff
@@ -0,0 +1,42 @@
+---
+import * as fs from "fs";
+import { kebabCasetoTitleCase } from "../../utils.ts";
+import { type LinkProps } from "../../types.ts";
+
+type SpokeDocCollectionType = "for-spoke-admins" | "for-spoke-texters";
+
+interface Props {
+    collectionType: SpokeDocCollectionType;
+}
+
+const { collectionType } = Astro.props;
+
+const DOCS_DIRECTORY = `${process.cwd()}/src/pages/docs/spoke`;
```

Comment:
This feels a little hacky, but it saves us needing to type up the list item for each doc ourselves. Since Astro is already creating the route for us,  and this is [fenced off to run server side anyway](https://docs.astro.build/en/core-concepts/astro-components/#the-component-script) anyway, it felt like an ok compromise, but definitely open to other thoughts!

Reply:
Also found that Astro supports similar access via [`Astro.glob()`](https://docs.astro.build/en/reference/api-reference/#astroglob) The Node methods look like a better fit here still since we're looking for file names, and [`Astro.glob()`](https://docs.astro.build/en/reference/api-reference/#astroglob) returns the file content itself